test/jasmine/tests/sankey_d3_sankey_circular_test.js

```diff
@@ -0,0 +1,76 @@
+// var Plotly = require('@lib/index');
+// var attributes = require('@src/traces/sankey/attributes');
+
+var d3SankeyCircular = require('d3-sankey-circular');
+var mockCircular = require('@mocks/sankey_circular.json');
+var convertToD3Sankey = require('@src/traces/sankey/convert-to-d3-sankey');
+
+describe('d3-sankey-ciruclar', function() {
```

Comment:
No need for two sankey test files. Can you move this `describe` to `sankey_test.js` and :hocho: this file?

Reply:
Done in 4539d285285ebb22d6d2ce9ddd300e9e1de03b60